app/templates/components/events/view/overview/general-info.hbs

```diff
@@ -0,0 +1,49 @@
+<div class="content">
+  <div class="header">{{t 'General info'}}</div>
+</div>
+<div class="content">
+  <div class="header"> {{t 'Your event is not live yet. Make it live now'}}</div>
+</div>
+<div class="content">
+  <table class="ui striped table">
+    <tbody>
+      <tr>
+        <th>{{t 'No. of Revisions'}}</th>
```

Comment:
Using a `th` inside a `tr` along with a `td`, results in bad styling. Instead make this and use `strong` to make the text bold